FB_RS485_EASTRON_SDM_POWER_MQTT: Adding support for SDM120

MichielVanwelsenaere opened the pull request:
## Pull Request Checklist
Check if the tasks below are relevant for your PR, to mark an item as completed use *[X]*.

* [X] Updated the PLCOpen XML export file (only relevant on code change).
* [X] Updated the CoDeSys V3 export file (only relevant on code change).
* [X] Updated or created any relevant documentation.

## Proposed Changes

- Adds support for the Eastron SDM120 on FB_RS485_EASTRON_SDM_POWER_MQTT

## Related issues

- Closes #125

meijer3 commented:
![image](https://user-images.githubusercontent.com/45625341/202913220-86526a0f-9f40-4019-94ba-d14e8711955d.png)
Yes, it works. Thanks!

meijer3 commented:
Only what confuses me is the home-assistant config

`      - topic: "Devices/PLC/House/Home/Out/RS485/FB_RS485_EASTRON_SDM_POWER_001/availability"
      - topic: "Devices/PLC/House/availability"`
The first topic has `Home`   in the topic. That is not the case in the/my POU.
I didnt check if it was on purpose, but I changed the config. It is also in your SDM220 markdown I noticed

MichielVanwelsenaere commented:
> Only what confuses me is the home-assistant config
> 
> ` - topic: "Devices/PLC/House/Home/Out/RS485/FB_RS485_EASTRON_SDM_POWER_001/availability" - topic: "Devices/PLC/House/availability"` The first topic has `Home` in the topic. That is not the case in the/my POU. I didnt check if it was on purpose, but I changed the config. It is also in your SDM220 markdown I noticed

good catch, a leftover from a refactor on the topic approach. Fixed!
